lib/xboxrt/libc_extensions/wchar.h

```diff
@@ -1,8 +1,11 @@
 #ifndef _XBOXRT_WCHAR_H
 #define _XBOXRT_WCHAR_H
 
+#include <assert.h>
 #include <stdint.h>
 #include <stddef.h>
+#include <time.h>
```

Comment:
Why?

Reply:
Because `wchar.h` needs to provide `tm`, which is declared in `time.h`. At the time I wrote this I thought this was the cleanest way to provide it (i.e. not having to deal with PDCLib internals here), however, reading the standard again, I realized that `wchar.h` only needs to provide an incomplete type, so I replaced this with `struct tm;`, for which I added a new commit.

Reply:
I didn't check the standard, but I'll trust your expertise.
Thanks for explaining this - this is really surprising to me and wasn't obvious earlier.

nit: the new line could have mentioned that this is required by the spec in a comment. However, I feel that it is implicitly documented in `git blame` now by explicitly adding it (in its own commit).